### DISCUSSION TITLE
GitHub Stats not showing Private Repo Statistics

### DISCUSSION THREAD

**Discussion by PeteVanBenthuysen** [anuraghazra/github-readme-stats#3973] [General]
I have everything properly setup. I forked https://github.com/PeteVanBenthuysen/github-readme-stats.git, created a key and gave it admin, repo, and user perms then I deployed my forked github-readme-stats onto vercel properly. I am getting an error that reads "No languages data". I want to keep my repos private but showcase my statistics on my github page. Here is the URL code I am using:

[![Top Langs](https://github-readme-stats-peter-vanbenthuysens-projects.vercel.app/api/top-langs/?username=PeteVanBenthuysen)](https://github.com/anuraghazra/github-readme-stats)

**Comment by iibrahemali** [comment]
Same Prob

**Comment by JayanshDevgan** [comment]
for that you have to fork this repo and push it with your own Personal Access token(PAT) with name is PAT_1 and the key as the token then use vercel to get stats by connecting your github to vercel and publishing your repo.

**Comment by JayanshDevgan** [comment]
I Problem is different i am not able to see my private repo commits on github stats
  src="my-vercel-url.vercel.app/api?username=jayanshdevgan&&show_icons=true&theme=react&border_radius=10"

**Comment by qwerty541** [comment]
Hello @PeteVanBenthuysen 

You need to set correct permissions for your personal access token in order to display your private contributions. Here is a link on the documentation with step by step guide for both classic and fine-grained token types https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#first-step-get-your-personal-access-token-pat
